AdyenCard/Components/Card/CardComponentDelegate.swift

```diff
@@ -10,20 +10,24 @@ import UIKit
 /// Delegate for observing user's activity on `CardComponent`.
 public protocol CardComponentDelegate: AnyObject {
 
-    /// Called when shopper enters PAN in `CardComponent`.
-    /// - Parameter value: Up to 6 first digits in entered PAN.
+    /// Called with the first 6 or 8 digits typed by the shopper in the PAN field in `CardComponent`.
```

Comment:
```suggestion
    /// Called with the first 6 or 11 digits typed by the shopper in the PAN field in `CardComponent`.
```

Reply:
it'll never be 11, BIN is max 8 digits :-)